dev.linwood.butterfly.appdata.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>dev.linwood.butterfly</id>
+	<name>Linwood Butterfly</name>
+	<developer_name>Linwood</developer_name>
+	<summary>Powerful, minimalistic, cross-platform, opensource note-taking app</summary>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>GPL-3.0-only</project_license>
+	<supports>
+	  <control>pointing</control>
+	  <control>keyboard</control>
+	  <control>touch</control>
+	</supports>
+	<recommends>
+		<control>pointing</control>
+		<control>keyboard</control>
+		<control>touch</control>
+	</recommends>
+	<developer_name>LinwoodCloud</developer_name>
+	<url type="homepage">https://github.com/LinwoodCloud/butterfly</url>
+	<description>
+		<p>Butterfly is a note app where your ideas come first. You can paint, add texts and export them easily on every device. This app works on android, windows, linux and in the web.</p>
+		<p>Features:</p>
+		<ul>
+			<li>Simple and intuitive: Every tool is on the right place. Open the app and start drawing. Change your tools by clicking on it.</li>
+			<li>Customizable: Change everything to your needs. Choose your custom color, create a palette and add your pages to the paper. The paper has an infinite size, perfect for your ideas and notes.</li>
+			<li>Your data will be saved locally: Everything will be saved local on your computer. Export the notes as image or as json files which can be opened on a different device.</li>
+		</ul>
+	</description>
+	<screenshots>
+		<screenshot type="default">
+			<image type="source">https://raw.githubusercontent.com/LinwoodCloud/butterfly/develop/docs/static/img/main.png</image>
+		</screenshot>
+	</screenshots>
+	<launchable type="desktop-id">dev.linwood.butterfly.desktop</launchable>
+	<releases>
+		<release version="1.0.0" date="2022-02-25" />
```

Comment:
You say here that your latest release is 1.0.0, but in the manifest your are downloading version 1.3.2